test/lib/core_extensions_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class CoreExtensionsTest <  ActiveSupport::TestCase
+  context 'string' do
+    test '#to_gb' do
+      value = "1024 MB"
+      to_gb_value = value.to_gb
+      assert_equal(1.0, to_gb_value, "Converted value should be 1.0")
+      assert(to_gb_value.is_a?(Float), "Converted value shoud be a float")
+    end
+
+    test '#to_utf8' do
+      value = Rails.public_path
+      to_utf8_value = value.to_utf8
+      assert_equal(Encoding::UTF_8, to_utf8_value.encoding)
```

Comment:
My point is that it ought to be OK to just expect `.encode('utf-8')` with the arguments you want to be called on whatever you call it on, exercising the method for real is something already tested [in ruby](https://github.com/ruby/ruby/blob/trunk/test/ruby/test_encoding.rb).

What you want to make sure in this test is that `.encode('utf-8')` is called upon the string, the actual conversion is performed & tested by Ruby.